src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -30,6 +34,7 @@ public class AddressBook implements ReadOnlyAddressBook {
      */
     {
         persons = new UniquePersonList();
```

Comment:
Are we gonna keep track of `persons` as well?

Reply:
Perhaps we can discuss our overall strategy on how and when we we will eliminate Person / AddressBook commands from the repo?